data/_ui/tooltips.txt

```diff
@@ -1092,23 +1092,242 @@ tip "This outfit is considered an atrocity."
 
 
 # Controls
-tip `Reverse`
-	`Turn opposite to the flight direction (or use reverse engines if you have them installed). Combine with <shift> key to stop your ship.`
+tip "Forward thrust"
+	`Accelerate your flagship forward.`
+
+tip "Turn left"
+	`Turn your flagship left.`
+
+tip "Turn right"
+	`Turn your flagship right.`
+
+tip "Reverse"
+	`Turn your flagship opposite its current direction of flight. If reverse thrusters are installed, accelerate backwards. With <shift>, stop your ship.`
+
+tip "Fire afterburner"
+	`Fire your flagship's afterburner, if one is installed.`
+
+tip "Auto steer"
+	`If the "Automatic aiming" setting is off, active automatic aiming when this key is pressed.`
+
+tip "Land on planet / station"
+	`Land on the selected planet. If no planet is selected, automatically selects a planet to land on.`
+
+tip "Initiate hyperspace jump"
+	`Initiate a hyperspace jump to follow your current travel plan. If you have no travel plan, initiates a jump nearest to your current facing angle.`
```

Comment:
Maybe it's worth mentioning that you can hold the key down to delay the jump?

Reply:
```suggestion
	`Initiate a hyperspace jump to follow your current travel plan. If you have no travel plan, initiates a jump nearest to your current facing angle. When held, commands your escorts to get ready to jump and only initiates the jump when released. With <shift>, sends the command to your fleet to get ready to jump and initiates the jump automatically when your entire fleet is ready instead of needing to hold the key.`
```